.bazelrc

```diff
@@ -14,6 +14,11 @@
 
 build --cxxopt=-std=c++17
 build --host_cxxopt=-std=c++17
+# llvm builds typically without rtti
```

Comment:
Consider linking to a reference for this, would be helpful for context and in case we need to adjust this in the future.